pom.xml

```diff
@@ -79,7 +79,7 @@
             <dependency>
                 <groupId>com.fasterxml.jackson.core</groupId>
                 <artifactId>jackson-databind</artifactId>
-                <version>${fasterxmlVersion}</version>
+                <version>2.9.9.3</version>
```

Comment:
Is jackson-databind not part of fasterxml and should be using fasterxmlVersion?
Why is databind 2.9.9.3 and the fasterxmlVersion 2.9.9

Reply:
jackson-databind alone has 2.9.9.3.. all other fasterxml version are 2.9.8 or 2.9.9.
Since it was asked to use the latest 2.9.9.3 for databind. I applied for jackson-databind and not the rest.

Reply:
Thanks for the clarification. Remember that when you make a pull request you can provide your own comments to explain things that might not be obvious to the reviewer.